spec/material-list-2.0.0.yaml

```diff
@@ -0,0 +1,179 @@
+openapi: 3.0.0
+info:
+  version: '2.0.0'
+  title: 'Collection List'
+  license:
+    name: 'GNU General Public License v3.0'
+    url: 'https://www.gnu.org/licenses/gpl-3.0.html'
+
+servers:
+  - url: https://prod.collectionlist.dandigbib.org
+    description: Production server (uses live data)
+  - url: https://test.collectionlist.dandigbib.org
+    description: Test server (uses test data)
+
+tags:
+  - name: 'List'
+    description: List handling
+  - name: 'Migrate'
+    description: Data migration
+security:
+  - BearerAuth: []
+paths:
+  /list/{listId}:
+    get:
+      operationId: getList
+      tags:
+        - List
+      description: 'Get list with collections.'
+      parameters:
+        - $ref: '#/components/parameters/listId'
+        - $ref: '#/components/parameters/collectionIds'
+      responses:
+        200:
+          description: 'The list data is returned.'
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/List'
+        404:
+          description: 'Unknown list.'
+        422:
+          description: 'Invalid collection id.'
+        default:
+          description: 'Unspecified error.'
+
+  /list/{listId}/{itemId}:
+    head:
+      operationId: itemAvailability
+      tags:
+        - List
+      description: 'Check existence of a collection on a list. To check multiple collections in one request, see the collection_ids query parameter on /list/{listId}.'
+      parameters:
+        - $ref: '#/components/parameters/listId'
+        - $ref: '#/components/parameters/itemId'
+      responses:
+        200:
+          description: 'The collection exists on the list.'
+        404:
+          description: 'The list or collection does not exist.'
+        422:
+          description: 'Invalid collection id.'
+        default:
+          description: 'Unspecified error.'
+    put:
+      operationId: addItem
+      tags:
+        - List
+      description: 'Add collection to the the list.'
+      parameters:
+        - $ref: '#/components/parameters/listId'
+        - $ref: '#/components/parameters/itemId'
+      responses:
+        201:
+          description: 'The collection was successfully added to the list.'
+        404:
+          description: 'Unknown list.'
+        422:
+          description: 'Invalid collection id.'
+        default:
+          description: 'Unspecified error.'
+    delete:
+      operationId: removeItem
+      tags:
+        - List
+      description: 'Delete collection from list.'
+      parameters:
+        - $ref: '#/components/parameters/listId'
+        - $ref: '#/components/parameters/itemId'
+      responses:
+        204:
+          description: 'Successfully removed.'
+        404:
+          description: 'Unknown list or collection.'
+        422:
+          description: 'Invalid collection id.'
+        default:
+          description: 'Unspecified error.'
+  /migrate/{legacyUserId}:
```

Comment:
This should be removed. The migration function will die before the move to v2.

Reply:
ok